tests/typecheck/contrib/admin/test_options.yml

```diff
@@ -12,7 +12,7 @@
       from django.db.models.query import QuerySet
       from django.utils.translation import gettext_lazy as _
 
-      def an_action(modeladmin: "A", request: HttpRequest, queryset: QuerySet) -> None:
+      def an_action(modeladmin: "A", request: HttpRequest, queryset: QuerySet[Model]) -> None:
```

Comment:
I recommend using some *real* model in tests, like `contrib.auth.models.User`, to be more similar to actual usage in the field.

Reply:
Oops, I don't think I'd meant to commit this. IIRC there's a test failure in both this file and `test_decorators`, at least partially because of the `display()` fixes, but I appreciate the suggestion and I'll do that when resolving the failing tests.